docs/api/serviceorchestration/orchestration-generic-http_dynamic.md

```diff
@@ -111,6 +109,16 @@ The service operation **responds** with the status code `200` if called successf
                }
             }
          ]
+         "authorizationTokens": {
+            "TIME_LIMITED_TOKEN_AUTH": {
+               "query-temperature": {
+                  "tokenType": "TIME_LIMITED_TOKEN_AUTH",
```

Comment:
This is bad, since this is a AuthorizationTokenType. The good value is TIME_LIMITED_TOKEN: